lib/lightning_web/live/run_live/run_viewer_live.ex

```diff
@@ -41,5 +42,6 @@ defmodule LightningWeb.RunLive.RunViewerLive do
       preload: :output_dataclip
     )
     |> Lightning.Repo.one()
+    |> Repo.preload(:log_lines)
```

Comment:
FYI I'm undecided whether we should preload logs or not, I see situations where it might be better to load them only when the logs tab is rendered. I've seen issues already with the way we are rendering them (i.e. browser crash) - so just a heads up that we need to revisit how and when we render log lines in LiveView.